services/catarse/spec/models/user_spec.rb

```diff
@@ -685,4 +685,44 @@ def create_contribution_with_payment(user, project, value, credits, payment_stat
       end
     end
   end
+
+  describe '#before_save' do
+    context 'when the user is created' do
+      context 'when the email references catarse' do
+        let(:user) { create(:user, email: 'example@catarse.me') }
+
+        it 'not created' do
```

Comment:
**Sugestão:** Acredito que ficaria mais completo adicionando um `it` com a mensagem de erro.